src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+
+
+
+Try out this Java tutorial to see what the New Relic platform can do with your OTLP data. You can quickly learn how to configure your OpenTelemetry SDK to optimize your observability experience in New Relic. You'll do all this using a demo Spring application that generates the nth number in the Fibonacci sequence.
+
+You'll find three tracks below to help you get acquainted with New Relic and OpenTelmentry. Try out the tracks that are interesting to you:
+
+* [Run our pre-instrumented program:](#pre-instrum) This is the fastest way to send some demo data to New Relic and see how it is displayed in the UI. This demo app has pre-loaded instrumentation and SDK configurations that follow our [best practices](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-overview/) to generate and export metrics, logs, and traces. You can inspect our code and apply relevant sections to your own apps.
+* [Use the OpenTelemetry Java agent:](#java-agent) Instead of using our pre-instrumentated demo app, you can use the OpenTelemetry Java agent to automatically monitor our demo app. You don't need to open the demo code and make any changes&mdash;just install and run the agent alongside the app.
+* [Instrument the demo app manually:](#manual-instrum) In this track, you'll roll up your sleeves and tinker with the engine of the car. This is the approach to take if you want to have the most control over what telemetry is reported and want to see details about how it's done. You'll manually insert instrumentation into our demo app to capture telemetry and you'll configure the SDK to send that data to New Relic.
+
+<Callout variant="tip">
+You have two choices for exporting data from your application to New Relic via OTLP:
+* Directly from your app
+* Via an OpenTelemetry collector
+
+This guide covers the first option. If you wish to export your data via a collector, check out this [documenation](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-basic/) for details.
+</Callout>
+## Requirements [#requirements]
+
+Before you get started, make sure you have the following:
+
+* A New Relic account: Sign up for a [free account](https://newrelic.com/signup) if you don't already have one.
+* Java 17 or higher: If you don't already have this, see [these download options](https://www.java.com/en/download/manual.jsp).
+
+## Run our pre-instrumented program [#pre-instrum]
+
+This is a great option if you want us do the instrumentation so you can quickly see what's it's like to send data to New Relic and view it in our UI.
+
+1. Clone the demo app and navigate to the following directory:
+    ```
+    git clone git@github.com:newrelic/newrelic-opentelemetry-examples.git
+    cd newrelic-opentelemetry-examples/getting-started-guides/java/Instrumented
+    ```
+2. In your terminal, set these two environment variables to send data to your New Relic account:
```

Comment:
the 'in your terminal' maybe should come in first step, little redundant in second step maybe

Reply:
@zuluecho9 Good catch!